source/frontend/src/features/properties/map/research/detail/ResearchSummaryView.tsx

```diff
@@ -90,6 +96,13 @@ const ResearchSummaryView: React.FunctionComponent<
           />
         ) : null}
       </StyledEditWrapper>
+      <Section header="Project">
+        <SectionField label="Ministry project">
+          {detail.researchFileProjects.map(formattedName => (
```

Comment:
seems a bit odd to display this section empty when there are no projects, but that is what the requirements ask for so /shrug